Cilsil.Test/Cilsil.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
For consistency, we should update [TestRunManager.cs](https://github.com/microsoft/infersharp/blob/64dc14a7542acfb042183482a22cabe4e66eeff8/Cilsil.Test/TestRunManager.cs#L31), [CONTRIBUTING](https://github.com/microsoft/infersharp/blob/98371c63612a22c35cc9148ebab3b149d989eab6/CONTRIBUTING.md#prerequisites), and [TestProject.csproj](https://github.com/microsoft/infersharp/blob/98371c63612a22c35cc9148ebab3b149d989eab6/Cilsil.Test/Assets/TestProject.csproj) to adopt net6.0